_posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md

```diff
@@ -0,0 +1,151 @@
+    ssl_fingerprint_verification_only: true
+    authentication:
+        mutual_tls:
+```
+
+In the example above, Data Prepper will discover other peers using DNS. It will perform a DNS query on the domain `my-data-prepper-cluster.production`.
+This DNS record should be an A record with a list of IP addresses for peers. The configuration uses a custom certificate and private key. 
+It performs host verification by checking the fingerprint of the certificate. And finally it configures each server to authenticate requests using 
+Mutual TLS (mTLS) to prevent tampering of data.
+
+## Directory structure
+
+Previously, Data Prepper was distributed as a single executable JAR file. This is simple and convenient, but also makes it difficult for Data Prepper 
+to include custom plugins. Data Prepper 2.0 introduces a change for it and now distributes the application in a bundled directory structure. 
+The new directory structure features a shell script to launch Data Prepper and dedicated subdirectories for JAR files, configurations, pipelines, logs, and more. 
+The directory structure looks like this:
+
+```
+data-prepper-2.0.0/
+  bin/
+    data-prepper                    # Shell script to run Data Prepper
+  config/
+    data-prepper-config.yaml        # The Data Prepper configuration file
+    log4j.properties                # Logging configuration
+  pipelines/                             # New directory for pipelines
+    trace-analytics.yaml
+    log-ingest.yaml
+  lib/
+    data-prepper-core.jar
+    ... any other jar files
+  logs/
+```
+
+With this change, a user can launch Data Prepper by simply running `bin/data-prepper`. No additional command line arguments or Java system property definitions 
+are required. Instead, the application will load configurations from `config/` subdirectory.
+
+Data Prepper will also read pipeline configurations from `pipelines/` subdirectory.  Users can now define pipelines across 
+multiple YAML files in the subdirectory, where each file contains the configuration for one or more pipelines. This will 
+allow users to keep their pipeline definitions distinct and thus more compact and focused. 
+
+## JSON & CSV parsing
+
+Many of our users have incoming data with embedded JSON or CSV fields. Now Data Prepper supports parsing either JSON or CSV.
+
+A common example of this is when one larger object includes a serialized JSON string. If your incoming event data has a 
+serialized JSON string, you can use the `parse_json` processor to extract the fields from the JSON into your event.
+
+Data Prepper can now import CSV or TSV formatted files from Amazon S3 sources. This is useful for systems like Amazon CloudFront 
+which write their access logs as TSV files. Now you can parse these logs using Data Prepper. Additionally, if your events have 
+CSV or TSV fields, Data Prepper has a `csv` processor which can create fields from your incoming CSV data.
+
+## Other improvements
```

Comment:
Sections edits:

---


Data Prepper 2.0 includes several other improvements. We want to highlight a few of them.

* The OpenSearch sink now supports any `create` actions for OpenSearch when writing or updating documents. Pipeline authors can configure their pipelines to create new documents when one does not already exist.
* The HTTP source now supports loading TLS/SSL credentials from either Amazon S3 or Amazon Certificate Manager. The OTel Trace Source supported these options; pipeline authors can now configure them for their log ingestion use cases.
* Data Prepper now requires Java 11, and the Docker image deploys with JDK 17.

Please see our release notes for a complete list.